### ISSUE TITLE
Sea Skulls [Yarr]

### ISSUE THREAD

**Issue by panpeus** [Rinnegatamante/YoYo-Loader-Vita-Compatibility#581]
NOTE: IF YOU'RE SUBMITTING A REPORT FOR A GAMING HAVING ISSUES, PLEASE PERFORM A RUN WITH DEBUG MODE ON AND POST THE RELATED LOG FILE HERE!

# Game summary
- Game name: [Sea Skulls](https://ranchogames.itch.io/seaskulls) 
- Game version: ?
- Asset swap with Cure Hunter

# YoYo Loader summary
- Version: e31c28

# Issues

- Purely a multiplayer game.
- Can't play game without second player

 # Screenshots
![](https://?)
![2022-06-19-003723-009006](https://user-images.githubusercontent.com/103394740/174494212-c52fe853-2c1f-4c08-9ef6-75a4bcc57594.jpg)

# Recommended labels

- In Game High
- Enable Fake Windows as Platform
[yyl.log](https://github.com/Rinnegatamante/YoYo-Loader-Vita-Compatibility/files/8936087/yyl.log)

**Comment by Rinnegatamante** [comment]
Welp, if the game is designed for multiplayer, setting it to ingame high cause you can't play it solo doesn't make sense. YoYo Loader supports multiple controllers over MiniVitaTV and PSTV

**Comment by panpeus** [comment]
You're right, thought I post it and see what you thought it. I found another game, that is also only multiplayer but didn't bother to report it, and that other one, has some major key binding issues.
